### ISSUE TITLE
How to add Scope params

### ISSUE THREAD

**Issue by mrjamesriley** [omniauth/omniauth-instagram#27]
I'm using Omniauth with Devise. It's not documented how to specify your scopes, and it does not appear to follow the convention used by facebook, github and others. So if it helps anyone, here's how you'd specify your desired permissions:

```  
config.omniauth :instagram, XXX, YYY, scope: %w(basic public_content follower_list)
```

The scope expects an array, or something that can be cast to an array.
Had me stumped for a moment till I dug through source.

**Comment by Electrosyrinx** [comment]
Thanks for posting, mrjamesriley.

**Comment by tmilewski** [comment]
Closed with #19
